### PR TITLE
Added M-Web-Server tests (Mumps)

### PR DESCRIPTION
**Pull request by sumeetchhetri**

### CODE REVIEW

**Comment by sumeetchhetri** [comment]
Ping @shabiel and @ChristopherEdwards
This is based on the fork of the original work/repository (https://github.com/shabiel/M-Web-Server)

**Comment by sumeetchhetri** [comment]
[MUMPS](https://en.wikipedia.org/wiki/MUMPS)

**Comment by shabiel** [comment]
That looks interesting! Thank you.

--Sam

File: .travis.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ env:
     - "TESTLANG=V"
     - "TESTLANG=Vala"
     - "TESTLANG=VB"
+    - "TESTLANG=Mumps"
```
Comment (nbrady-techempower) [inline .travis.yml:93]: Could you put this after `Lua`? Keeping it alphabetical.

Comment (sumeetchhetri) [reply .travis.yml]: Done

**Pull request merged**